package-lock.json

```diff
@@ -3046,7 +3046,7 @@
     },
     "p-is-promise": {
       "version": "1.1.0",
-      "resolved": "https://registry.npmjs.org/p-is-promise/-/p-is-promise-1.1.0.tgz",
+      "resolved": "http://registry.npmjs.org/p-is-promise/-/p-is-promise-1.1.0.tgz",
```

Comment:
I still wonder why this keeps happening. Seems like some issue with npm

Reply:
I have npm `6.4.1`

Reply:
Happens on my machine as well, not sure why. A while ago I updated all of the urls to https manually.

Reply:
May be npm just try access to `https` first and after some short timeout fallback to `http`? But I don't think npm verify links, this will be pretty inefficient

Reply:
It seems this old problem: https://github.com/npm/npm/issues/20719